app/routes.py

```diff
@@ -361,3 +373,88 @@ def ags_export(bgs_loca_id: int = ags_export_query):
     headers = {'Content-Disposition': f'attachment; filename="{filename}"'}
 
     return Response(response.content, headers=headers, media_type='application/x-zip-compressed')
+
+
+@router.get("/ags_export_by_polygon/",
+            tags=["ags_export_by_polygon"],
+            summary="Export a number of boreholes in .ags format",
+            description=("Export a number of boreholes in .ags format from AGS data "
+                         "held by the National Geoscience Data Centre."),
+            include_in_schema=True,
+            response_class=Response,
+            responses=zip_ags_responses)
+def ags_export_by_polygon(polygon: str = polygon_query):
+    """
+    Export the boreholes in .ags format from AGS data held by the National Geoscience Data Centre,
+    that are bounded by the polygon. If there are more than 10 boreholes return an error
+    :param polygon: A polygon in Well Known Text.
+    :type polygon: str
+    :return: A response containing a .zip file with the exported borehole data.
+    :rtype: Response
+    :raises HTTPException 404: If there are no boreholes or more than 10 boreholes in the polygon.
```

Comment:
I am treating these two cases as 404s, returning errors that effectively say no values can be returned. This matches the unknown ID in the `ags_export` endpoint.